app/tests/test_app.py

```diff
@@ -0,0 +1,48 @@
+import unittest
+from app.app import app
+from app.config import API_VERSION
+
+
+class TestApp(unittest.TestCase):
```

Comment:
All these tests are probably a great case for mocking out the 'service' layer and only care about responses
But it's a nightmare in Python so yeah